src/coins.cpp

```diff
@@ -1093,6 +1093,7 @@ std::optional<UnsatisfiedShieldedReq> CCoinsViewCache::HaveShieldedRequirements(
 
 bool CCoinsViewCache::HaveInputs(const CTransaction& tx) const
 {
+    // *TCR:bad-txns-inputs-missingorspent
```

Comment:
```suggestion
    // https://p.z.cash/TCR:bad-txns-inputs-missingorspent?partial
```
(see https://github.com/zcash/orchard/pull/321 for other examples.)